src/core/core/rpc_engine.cpp

```diff
@@ -850,37 +850,35 @@ namespace dsn {
                             {
                                 resolver->on_access_failure(req2->header->gpid.u.partition_index, err);
 
-                                /*
-                                // TODO(qinzuoyan): because the user will fetch reply states from this response_task 'call',
-                                // so it's useless to create another response_task, which will not be handled by user.
-
                                 // still got time, retry
                                 uint64_t nms = dsn_now_ms();
-                                if (nms + 1000 < timeout_ts_ms)
+                                if (nms + 10 < timeout_ts_ms)
```

Comment:
10ms may be too short~
for 1 second timeout, it will retry for 100 times, and generate too many error logs.